flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/operations/TableOperationTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.operations;
+
+import org.apache.flink.api.common.typeinfo.Types;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.table.expressions.FieldReferenceExpression;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for describing {@link TableOperation}s.
+ */
+public class TableOperationTest {
+
+	@Test
+	public void testSummaryString() {
+		TableSchema schema = TableSchema.builder().field("a", DataTypes.INT()).build();
+
+		ProjectTableOperation tableOperation = new ProjectTableOperation(
+			Collections.singletonList(new FieldReferenceExpression("a", Types.INT, 0, 0)),
+			new CatalogTableOperation(
+				Arrays.asList("cat1", "db1", "tab1"),
+				schema), schema);
+
+		SetTableOperation unionTableOperation = new SetTableOperation(
+			tableOperation,
+			tableOperation,
+			SetTableOperation.SetTableOperationType.UNION,
+			true);
+
+		assertEquals("UNION: (all: [true])\n" +
```

Comment:
Why is "Union" in upper cases?

Reply:
I used the name from the enum. I updated it to have it properly formatted.